src/main/java/appeng/parts/p2p/PartP2POpenComputers.java

```diff
@@ -80,8 +72,24 @@ public PartP2POpenComputers( final ItemStack is )
 		{
 			this.node = null; // to satisfy final
 		}
+	}
+
+	@MENetworkEventSubscribe
+	public void changeStateA( final MENetworkBootingStatusChange bs )
+	{
+		this.updateConnections();
+	}
 
-		this.updateCallback = new UpdateCallback();
+	@MENetworkEventSubscribe
+	public void changeStateB( final MENetworkChannelsChanged bs )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "bs". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)